### PR TITLE
remove e2e-tests

### PR DESCRIPTION
**Pull request by maorfr**
part of https://issues.redhat.com/browse/APPSRE-7589

### CODE REVIEW

**Review by OliviaHY — APPROVED** [review]

File: reconcile/gql_definitions/common/clusters.gql [hunk 1]
```diff
@@ -13,7 +13,7 @@ query Clusters($name: String) {
     managedClusterRoles
     insecureSkipTLSVerify
     jumpHost {
-      ... CommonJumphostFields
+      ...CommonJumphostFields
```
Comment (rporres) [inline reconcile/gql_definitions/common/clusters.gql:16]: Please avoid linting changes 🙏

**Review by rporres — COMMENTED** [review]
Can you take a look to the amount of linting changes that doesn't belong to this PR? They are certainly weird, as if it was a GH artifact

File: reconcile/gql_definitions/common/clusters.gql (hunk 1 above)
Comment (maorfr) [reply reconcile/gql_definitions/common/clusters.gql]: that wasn't intentional :sweat_smile:

Comment (maorfr) [reply reconcile/gql_definitions/common/clusters.gql]: it was "prettier": prettier.io

**Pull request merged**